### PR TITLE
CCD-1223: move befta test definitions to their new repo

### PR DESCRIPTION
**Pull request by MSancaktutar**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CCD-1223

### Change description ###
use befta fw and ccd definitions from separate artefacts

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[X] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    description = "Runs DefinitionConverter for Excel to JSON"
+
+    doFirst() {
+        fileTree("${project.projectDir}/src/main/resources/uk/gov/hmcts/befta/dse/ccd/definitions/excel").matching {
+            include "*.xlsx"
+            exclude "~*.xlsx"
+        }.visit { FileVisitDetails details ->
+            def name = details.getName()
+            def inputPath = "${project.projectDir}/src/main/resources/uk/gov/hmcts/befta/dse/ccd/definitions/excel/${name}"
+            def outputFolderPath = "${project.projectDir}/src/main/resources/uk/gov/hmcts/befta/dse/ccd/definitions/valid/"
```
Comment (mattnayler) [inline build.gradle:90]: The three paths to the excel folder and output JSON (lines 84, 89-90) need updating to reflect the new locations. 

To test the updated config:
* delete the output folder (i.e. `src/main/resources/uk/gov/hmcts/ccd/test_definitions/valid/`)
* run  `./gradlew definitionsToJson`
* confirm regenerated output folder still matches previous JSON files already in git.

> NB: the **DefinitionStore** also contains a number of references to this folder: e.g. inside `DefinitionStoreTestAutomationAdapter` and throughout its scenario data files.  These will need changing when it is switched over to load data from **ccd-test-definitions**.

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    id 'java-library'
+    id 'maven-publish'
+    id 'maven'
+}
+
+apply plugin: 'java'
+
+def buildNumber = System.getenv("RELEASE_VERSION")?.replace("refs/tags/", "") ?: "7.0.0-SNAPSHOT"
+
+group 'uk.gov.hmcts.ccd'
```
[the remark this answers is not in the document]
Comment (MSancaktutar) [reply build.gradle:11]: befta-fw is meant to be a hmcts-generic artefact, while ccd definitions are meant only for ccd - hence the difference in the group.

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.3-all.zip
```
[the remark this answers is not in the document]
Comment (MSancaktutar) [reply gradle/wrapper/gradle-wrapper.properties:3]: Could be, but I'd suggest to keep this to the current standard inherited from befta-fw, just to be able to keep this minimal for its own purpose.

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+/bin/
```
[the remark this answers is not in the document]
Comment (MSancaktutar) [reply .gitignore:25]: done

File: build.gradle (hunk 1 above)
Comment (MSancaktutar) [reply build.gradle]: done

File: build.gradle (hunk 2 above)
[the remark this answers is not in the document]
Comment (MSancaktutar) [reply build.gradle]: But, to keep consistent with local builds, I've updated the groups accordingly.

**Review by mattnayler — APPROVED** [review]
LGTM 👍

**Pull request merged**